docs/requirements.txt

```diff
@@ -1,4 +1,5 @@
 sphinx==3.5.1
```

Comment:
I think it should also be fine just updating sphinx to the latest version.

Reply:
So far I'm trying to minimize changes, it's already more challenging than expected :)

Reply:
Okay, hit the old CSS issues coming from old Sphinx + docutils 0.17. Upgrading Sphinx.